include/enums.h

```diff
@@ -1457,6 +1458,7 @@ enum Triggers {
 };
 
 enum Buttons {
+    BUTTON_NONE         = 0x00000000,
```

Comment:
🤨

Reply:
what, you've never used the none-button?